crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -2629,6 +2629,10 @@ where
                     }
                 }
 
+                // Mine
+                if self.settings.rules.enabled(Rule::BadSuperCall) {
+                    pylint::rules::bad_super_call(self, expr, func, args);
+                }
```

Comment:
Temporary place for it.

Might keep it here, but if I do I will comment `//pylint` here